### PR TITLE
Making AOR Reports more responsive

### PR DESCRIPTION
**Pull request by daniel-samson**
## Description

In order to make the reports with SuiteP on the small desktop. I have moved the module tree into a panel under the basic panel. This gives more space for the line items in the fields / conditions / charts. I have also restyled reports to look like the new changes in workflow.  Plus it now works on an iPad.

## Motivation and Context

* Makes the reports more usable.

## How To Test This

* Compile the SASS
* Create a report using a small desktop 1024x768

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: modules/AOR_Fields/fieldLines.js [hunk 1]
```diff
@@ -304,24 +302,24 @@ function insertFieldLine(){
...
+        a.innerHTML = "<button type='button' id='aor_fields_delete_line" + fieldln + "' class='button aor-remove-btn' value='' tabindex='116' onclick='markFieldLineDeleted(" + fieldln + ")'><img src='themes/"+SUGAR.themes.theme_name+"/images/id-ff-remove-nobg.png' alt=''></button><br>";
         a.innerHTML += "<input type='hidden' name='aor_fields_deleted[" + fieldln + "]' id='aor_fields_deleted" + fieldln + "' value='0'><input type='hidden' name='aor_fields_id[" + fieldln + "]' id='aor_fields_id" + fieldln + "' value=''>";
     } else{
         a.innerHTML = fieldln +1;
     }
-    a.innerHTML += "<input type='hidden' name='aor_fields_field_order[" + fieldln + "]' id='aor_fields_field_order" + fieldln + "' value='"+fieldln+"'>"
-    a.style.width = '5%';
+  a.innerHTML += "<input type='hidden' name='aor_fields_field_order[" + fieldln + "]' id='aor_fields_field_order" + fieldln + "' value='" + fieldln + "'>";
+  a.style.width = ''; // 5%
```
Comment (mattlorimer) [inline modules/AOR_Fields/fieldLines.js]: // 5% needed? do we need to set empty?

File: themes/SuiteP/css/editview.scss [hunk 2]
```diff
@@ -2631,10 +2631,212 @@ input#btn_ActionLine.button[type="button"] {
...
+  border-top-right-radius: 0;
+  border-top: none;
+  border-bottom: 4px solid $main-alternate-bg;
+}
+
+[aor-field-line-body] td  {
+  background-color: $detail-view-field-bg;
+  border-top: 2px solid $main-alternate-bg;
+  border-bottom: 2px solid $main-alternate-bg;
+  border-top-left-radius: 4px;
```
Comment (mattlorimer) [inline themes/SuiteP/css/editview.scss]: border-radius : 4px? all needed?

File: modules/AOR_Conditions/conditionLines.js [hunk 3]
```diff
@@ -507,12 +507,12 @@ function insertConditionLine(condition){
...
         var a = x.insertCell(nxtCell++);
         if(action_sugar_grp1 == 'EditView'){
-            a.innerHTML = "<button type='button' id='aor_conditions_delete_line" + condln + "' class='button' value='' tabindex='116' onclick='markConditionLineDeleted(" + condln + ")'><img src='themes/default/images/id-ff-remove-nobg.png' alt=''></button><br>";
+            a.innerHTML = "<button type='button' id='aor_conditions_delete_line" + condln + "' class='button' value='' tabindex='116' onclick='markConditionLineDeleted(" + condln + ")'><img src='themes/" + SUGAR.themes.theme_name + "/images/id-ff-remove-nobg.png' alt=''></button><br>";
             a.innerHTML += "<input type='hidden' name='aor_conditions_deleted[" + condln + "]' id='aor_conditions_deleted" + condln + "' value='0'><input type='hidden' name='aor_conditions_id[" + condln + "]' id='aor_conditions_id" + condln + "' value=''>";
         } else{
             a.innerHTML = condln +1 + "<input class='aor_conditions_id' type='hidden' name='aor_conditions_id[" + condln + "]' id='aor_conditions_id" + condln + "' value=''>";
         }
-        a.style.width = '5%';
+        a.style.width = ''; // 5%
```
Comment (mattlorimer) [inline modules/AOR_Conditions/conditionLines.js]: empty?

File: modules/AOR_Fields/fieldLines.js (hunk 1 above)
Comment (daniel-samson) [inline modules/AOR_Fields/fieldLines.js]: Removed lines.

File: modules/AOR_Conditions/conditionLines.js (hunk 3 above)
Comment (daniel-samson) [inline modules/AOR_Conditions/conditionLines.js]: Removed lines

File: themes/SuiteP/css/editview.scss (hunk 2 above)
Comment (daniel-samson) [inline themes/SuiteP/css/editview.scss]: The border-radius is needed to keep to the same aesthetic as SuiteP - which employ rounded corners for more elements. It's a design choice. Plus it helps to visually separate the lines.

**Comment by daniel-samson** [comment]
Fixed Drag and Drop issue.

File: themes/SuiteP/css/editview.scss (hunk 2 above)
Comment (daniel-samson) [inline themes/SuiteP/css/editview.scss]: changed border-radius: 4px

**Pull request closed without merging**